code/modules/mapping/mapping_helpers.dm

```diff
@@ -1359,3 +1359,35 @@ INITIALIZE_IMMEDIATE(/obj/effect/mapping_helpers/no_lava)
 
 	engraved_wall.AddComponent(/datum/component/engraved, engraving["story"], FALSE, engraving["story_value"])
 	qdel(src)
+
+/obj/effect/mapping_helpers/emergency_shuttle_event_override
+	name = "emergency shuttle events override"
+	icon_state = "shuttleeventoverride"
+	late = TRUE
+	/// List of event paths to run
+	var/list/events_override = list()
+
+/obj/effect/mapping_helpers/emergency_shuttle_event_override/LateInitialize()
+	var/obj/docking_port/mobile/port = SSshuttle.emergency
+	if(!port)
+		qdel(src)
+		return
+	port.event_list.Cut()
```

Comment:
This doesn't work, templates are loaded before the random events are. IMO you don't need to account for this, the occasional rare meteor carp slaughter seems funny. Otherwise you could just block the shuttles random event gen if the list already contains something